packages/web-backend/src/Users/index.ts

```
@@ -0,0 +1,11 @@
+import { Effect } from "effect";
+
+// import { UsersRepo } from "./UsersRepo";
+
+export class Users extends Effect.Service<Users>()("Users", {
+  effect: Effect.gen(function* () {
+    // const repo = yield* UsersRepo;
+
+    return {};
+  }),
```

Biome (2.1.2)

[error] This generator function doesn't contain yield.



(lint/correctness/useYield)